### PR TITLE
Feature/forward mobtest issues

### PR DESCRIPTION
**Pull request by diasdm**
Implements the forwarding of mobtest issues to SonarQube.

### CODE REVIEW

**Comment by diasdm** [comment]
@rodrigo-fittipaldi if I understand correctly, this should have very low impact since most repositories don't have any quality gate right?

**Comment by diasdm** [comment]
Tested here:
https://github.com/MOV-AI/movai_ot_ottobo/pull/34
Build here:
https://sonarqube.aws.cloud.mov.ai/project/issues?id=movai_ot_ottobo&open=AYALXe1g5P6Cc10CQiae&resolved=false&sinceLeakPeriod=true&types=BUG

**Review by AlexFernandes-MOVAI — APPROVED** [review]
Really nice, first time I see it working.

One remark, on Sonar the report of issues is done through the name ENGINEID

Is there a way to give it a better name like MOBTEST, MOVAI_METADATA or something of your choice

https://sonarqube.aws.cloud.mov.ai/project/issues?id=movai_ot_ottobo&open=AYALXe1F5P6Cc10CQiac&resolved=false&rules=external_engineId%3ADocumentation+-+Description+missing&scopes=MAIN

**Comment by diasdm** [comment]
Issue opened: https://github.com/MOV-AI/metadata-static-analysis/issues/28

**Review by duartecoelhomovai — APPROVED** [review]

**Comment by duartecoelhomovai** [comment]
Was this PR forgotten? @davidmovai can it be merged?

File: .github/workflows/ros-workflow.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
 
     - name: Mobtest validations
       run: |
-        mobtest $(pwd)
+        mobtest --sonar_issues_path ./mobtest-issues.json .
```
Comment (duartecoelhomovai) [inline .github/workflows/ros-workflow.yml:102]: if that json file does not exist, does mobtest blow up?

Comment (diasdm) [reply .github/workflows/ros-workflow.yml:102]: That is the file to which mobtest writes the results, it doesn't matter if exists or not :)

**Review by rodrigo-fittipaldi — APPROVED** [review]

**Pull request merged**